frontend/cypress/e2e/beta/beta-model.cy.ts

```diff
@@ -13,7 +13,7 @@ describe('Beta create new model', () => {
     cy.get('[data-test=teamSelector]').click()
     cy.get('[role=presentation]').click()
     cy.get('[data-test=modelSelector]').type('test model')
-    cy.get('[data-test=modelDescription]').type('test description')
+    cy.get('[aria-label=modelDescription]').type('test description')
```

Comment:
Is there a reason we're using the aria-label and not the data-test property?

Reply:
I have altered it now so the tag is data-test rather than aria-label to keep it consistent with the other elements on the screen